models/base.rb

```diff
@@ -7,7 +7,23 @@ def pp_created_at
     created_at&.strftime('%Y-%m-%d')
   end
 
+  def pp_created_time
```

Comment:
let's not do this. get rid of all of these and just return created_at and updated_at as unix time stamps, time.to_i or something like that, and just render the time on the client, i updated chat to do the same

Reply:
so just 2 defs there or scrap it altogether?